helm-chart/renku/values.yaml

```diff
@@ -326,67 +326,101 @@ redis:
   ### https://github.com/bitnami/charts/tree/master/bitnami/redis         ###
   ###########################################################################
 
+  # the values below were taken from the default values for the chart
+  # and adapted to our context. The following contains a mix of specific
+  # customisations plus some higher level defaults which have been
+  # retained here so someone can get some overview of what this values
+  # file can do. 
+
   fullnameOverride: renku-redis
-  redisPort: 6379
-
-  ## config for newer redis versions, here for later reference
-  # replicas:
-  #   replicaCount: 2
-  # auth:
-  #   enabled: true
-  #   sentinel: true
-  #   existingSecret: redis-secret
-  #   existingSecretPasswordKey: redis-password
-
-  usePassword: true
-  existingSecret: redis-secret
-  existingSecretPasswordKey: redis-password
-
-  master:
-    # Fixed redis bug prevents us from using this before
-    # we bump the redis chart version
-    # spreadConstraints:
-    #   - maxSkew: 1
-    #     topologyKey: node
-    #     whenUnsatisfiable: DoNotSchedule
-    #     labelSelector:
-    #       matchLabels:
-    #         app: redis
-    persistence:
-      enabled: false
+
+  # default values in this section
+  image:
+    registry: docker.io
+    repository: bitnami/redis
+    tag: 7.0.7-debian-11-r7
+    pullPolicy: IfNotPresent
+
+  # replication is the default; standalone is also an option but this results in
+  # no read only replicas
+  architecture: replication
```

Comment:
shouldn't the default values be a minimal set that will get the thing up and running? Replication, sentinels, etc is more suitable for production deployments and brings additional complexity/requirements

Reply:
Iiuc, sentinels were enabled in the older stuff by default. The redis helm chart defaults to 3 replicas and the resource requirements are typically small (cpu: 250m, memory: 256Mi are examples in the helm chart). Given that the aggregate requirements of redis overall are modest, I think we should just go with this. I can add a comment to this effect if we wanna go with this approach.

Reply:
ok sounds good!